carp.test/src/commonMain/kotlin/dk/cachet/carp/test/runBlockingTest.kt

```diff
@@ -0,0 +1,3 @@
+package dk.cachet.carp.test
+
+expect fun runBlockingTest(block: suspend () -> Unit)
```

Comment:
Very cool solution. Doesn't seem to cause too many problems this way. Minor note: carp Kotlin code takes spaces in parenthesis.

Reply:
Missed the spaces in parenthesis style. Do you have a style guide written somewhere? I dont see one in your repo. Maybe an automatically apply style guide to PRs could be added?

Reply:
It's in my head. =)